### DISCUSSION TITLE
Swift runtime failure in PKCS7Padding.swift after version 1.4.0 update

### DISCUSSION THREAD

**Discussion by alperkayabasi** [krzyzanowskim/CryptoSwift#934] [General]
Hello,
I am using this framework for nearly three years. It was working fine until version 1.4.0 but now I am getting swift runtime failure in PKCS7Padding.swift "add" function.

**Reproduce**
Steps to reproduce:
1. I have a string aes key.
2. I am converting this string value to hexadecimal Data.
3. Then I am trying to create an AES object with this "hexKey" , the app crashes with this failure.

It's working in version 1.3.8, there is no failure. Only in 1.4.0.

`AES(key: Array(hexKey), blockMode: ECB(), padding: .pkcs7)`

![Screen Shot 2021-05-03 at 13 24 23](https://user-images.githubusercontent.com/2501751/116893153-512dc700-ac39-11eb-9055-5f8ea289e6ed.png)

**Comment by krzyzanowskim** [comment]
Would you mind post (or attach) a sample I could run to reproduce this very issue? What is the device it runs on?
